Makefile

```diff
@@ -104,6 +104,12 @@ define format_only
     @echo "# done"
 endef
 
+# run all code quality tools in parallel
+define metalinter_only
+    @docker build -t vpp_metalinter -f ./docker/development/Dockerfile.metalinter .
+    @docker run --rm vpp_metalinter
```

Comment:
can we instead build the vpp metalinter and just run it ?

Reply:
I'm not sure what you mean... these two lines are building and running it 🤔

Reply:
i mean we don't build it on the fly, it doesn't look we will need to rebuild it everytime, so why not we just push it to dockerhub and pull it every time ?

Reply:
It has to `COPY` in the code every time so we do need to build it every time.  All of the major parts are cached, though.  It's just the `COPY` + `WORKDIR` + `ENTRYPOINT` that gets built.  This is how all the "checks" Dockerfiles work across all our repos (at least the ones I've been involved with 😄 ).

The other option is to use bindmounts, but those have the disadvantages we already discussed so I think we should continue to avoid them in general.

Reply:
fair enough